astropy/wcs/wcsapi/wrappers/base.py

```diff
@@ -0,0 +1,77 @@
+import abc
+
+from astropy.wcs.wcsapi import BaseLowLevelWCS, wcs_info_str
+
+
+class BaseWCSWrapper(BaseLowLevelWCS, metaclass=abc.ABCMeta):
+    """
+    A base wrapper class for things that modify Low Level WCSes.
+
+    This wrapper implements a transparent wrapper to many of the properties,
+    with the idea that not all of them would need to be overridden in your
+    wrapper, but some probably will.
+    """
+    def __init__(self, wcs, *args, **kwargs):
```

Comment:
There's no `Parameters` section. In different examples I've seen `wcs` as a WCS object or a list of WCSs.